pkgs/development/python-modules/tpm2-pytss/default.nix

```diff
@@ -7,22 +7,17 @@
 buildPythonPackage rec {
   pname = "tpm2-pytss";
 
-  # Last version on github is 0.2.4, but it looks
-  # like a mistake (it's missing commits from 0.1.9)
   version = "1.1.0";
   disabled = pythonOlder "3.5";
+  format = "pyproject";
 
   src = fetchPypi {
     inherit pname version;
     sha256 = "sha256-O0d1b99/V8b3embg8veerTrJGSVb/prlPVb7qSHErdQ=";
   };
-  postPatch = ''
-    substituteInPlace tpm2_pytss/config.py --replace \
-      'SYSCONFDIR = CONFIG.get("sysconfdir", "/etc")' \
-      'SYSCONFDIR = "${tpm2-tss}/etc"'
-  '';
 
-  nativeBuildInputs = [ pkg-config swig ];
+  buildInputs = with python3.pkgs; [ pkgconfig asn1crypto ];
+  nativeBuildInputs = [ swig ];
```

Comment:
Pkgconfig from python or from pkgs?

Reply:
`pkgconfig` it's a python package required here during build time... Ummm, what's the difference between python3.pkgs and python3Packages?

Reply:
None, its just an alias.